frontend/app/components/forms/login-form.js

```diff
@@ -19,6 +21,25 @@ export default Component.extend({
 
     logOut() {
       this.get('session').close();
+    },
+    show1() {
```

Comment:
can we please have a better sets of names for the functions so that the readability of the project is maintained.
Thank you